packages/cli/src/commands/init/init.js

```diff
@@ -171,7 +192,7 @@ function createProject(projectName: string, options: Options, version: string) {
 }
 
 export default (async function initialize(
-  [projectName]: Array<string>,
+  [projectName, customProjectPath]: Array<string>,
```

Comment:
Maybe `projectPath` and default value `.` (because by default, I guess we create in CWD?)

Reply:
If you agree with my comment https://github.com/react-native-community/cli/pull/365/files#r281157763, I believe we should just do:
```
const projectPath = customProjectPath || projectName
```